### ISSUE TITLE
The Institution Problem

### ISSUE THREAD

**Issue by Zephyrum** [UndeadWizard/Occultus-Insaniam#23]
Summarizing it, in Europa Universalis 4, Institutions are one of the leading reasons of technological disparity. For a game in the new continents to be playable, it's necessary that we ensure the answer to all "can (continent) get (institution)?" is a *Yes*.

Overarching methods that allow them to bypass a No are not really good options:
-Bordering someone with the institution is difficult for those isolated landmasses;
-Develop to embrace is never done by the AI;
-Knowledge share requires a DLC.

**Feudalism**
*Methods*: Not a Horde nor in the New World
Atlantis: Yes. They are not hordes, and start with it anyway.
Lemuria: Yes. Borders India that has it (usually). Likely has it too.
Mu: Yes. Nations within Mu already have it.

**Renaissance**
*Methods*: High development European province, faster if in Italy or Belgium 
Atlantis: No.
Lemuria: Yes, through India, will take very long though.
Mu: No.

**Colonialism**
*Methods*: Have a colonial nation under you.
Atlantis: Yes, albeit unlikely. Specially so for Antillia. Requires American colonies.
Lemuria: Yes, but very unlikely. Lemurians are unlikely to take the colonial path, Australia being the only one in range at those techs. Alternatively, through India.
Mu: Yes, albeit unlikely. If unchanged, same thing as Atlantis - needs the Americas.

**Printing Press**
*Methods*: Protestant, Reformed or Anglican country/province, or high development European province.
Atlantis: No, unless a decision to convert to Old World Christianity is enabled, then Heptarchics have a small chance to go down a branch of Protestantism.
Lemuria: Yes, but very slowly, through India.
Mu: No.

**Global Trade**
*Methods*: Control level 2 or 3 Center of Trade, have high Trade Power in province, have Printing Press
Atlantis: Yes.
Lemuria: Yes.
Mu: Yes.

**Manufactories**
*Methods*: Have manufactories in your provinces.
Atlantis: Yes.
Lemuria: Yes.
Mu: Yes.

**Enlightenment**
*Methods*: Parliament seats or Universities in provinces, have Innovative ideas, employ specific advisors.
Atlantis: Yes.
Lemuria: Yes.
Mu: Yes.

**Comment by Zephyrum** [comment]
Odd cases: Thule and Anthropofagoi. I don't find it troublesome that Anthropofagoi doesn't see institution spread due to it being, after all, an isolated cannibal island tribe. Thule, however, is essentially European and papar.

Because Thule is in Europe, they get passive spread from development for the Renaissance, have a sort of good position to colonize and gain Colonialism (AI likely to have trouble here), but is likely to lag behind on the Press, as they are unlikely to convert to a protestant branch. Developing themselves to 20 means passive spread in Thule, the likelyhood of that happening is unknown. If anything, they get passive spread when their diplomatic tech hits 15.

Further institutions just show up so long as they are doing something in their island.

**Comment by Zephyrum** [comment]
Identifying the troublesome ones is easy. Renassaince, Colonialism and the Press are the big three troublesome ones. Yet they're all some degree of manageable.

**Atlantis**

Atlantis for the most part outdoes Portugal and Spain in availability of colonies from the getgo, but this is hard to work with, as we'd be relying solely on the AI. My best guess is that for the Renaissance, Atlantis could have events providing spread to countries; Western European states that have the institution and can see Atlantis would be able to trigger an event for the foremost power of Atlantis, or maybe the westernmost, offering to share Renaissance knowledge. The same would happen in Antillia.

Colonialism would perhaps work similarly, or maybe even more simply; Atlantis and Antillia are both already "colonial" in history (Phoenicians, Greeks, Latins, Visigoths), so they'd have an event giving them passive spread soon after the birth of the institution.

Nothing really comes to mind for the Press other than making one monthly spread modifier that Atlanteans can make use of.

**Lemuria**

Increased institution spread speed in the continent by some 25% passively would make the continent embrace the early institutions about as quickly as China does. Not that bad of an imbalancement on it's own.

**Mu**

Colonial nations, and enabling Colonialism growth for nations with colonies in it. China and company will be able to spread the institution. Perhaps the Chinese exile could be a source of Colonialism, since they are technically a colony already - an event set to spawn 5 years after the institution spawns in their Capital would spread it to the whole continent.

The Renaissance and Press are tougher to handle. The easy way it to add a decision or event of sorts that enables it's spawning in the region, but I can't find an in-universe way to explain those.

**Comment by Zephyrum** [comment]
Atlantean Renaissance
Fires only once, for the highest development Atlantean state that is not a subject

Conditions:
-Is in Atlantean Tech Group
-Capital is in Atlantis
-Year is at least 1470
-Can see a province in Western Europe
-Atlantean Empire tag does not exist
-Does not have global flag "atlantean_renaissance"

Mean Time to Happen: 30 years
Lower if:
-Has 150 development
-Has 200 development
-Is a Great Power (requires RoM)
-Allied to Western tech group country
-Has a Western tech group subject
-Has a province with more than 25 development
-Has a Level 3 Center of Trade (requires Dharma)

"For ages, Atlantis has not seen grand architecture or artistic expansion, as the Imperial power collapsed and very few were in good enough condition to bring forth grand works, specially due to the lack of centralized funding. But this has been slowly changing - the local warlords seek to legitimize themselves, either as local rulers or candidates for Imperatorship. Some others just want to show off their growing wealth, reminding their peasantry of their position. Regardless of motive, local investments on grander buildings and exquisite pieces of art, from paintings to statues, are a growing trend in Atlantean society. European explorers have catalyzed that twofold by mentioning grand works of their homelands, some of which we are well aware of, albeit always considered mythical, as well as giving us stunned reactions to our prodigious artists, claiming it reminds them of the Greco-Roman styles whose nostalgia has been a growing trend in Europe."

Only Option: "It seems we're experiencing a cultural rebirth!"
+50% Renaissance progress in capital area (including provinces owned by other countries)
+50% Renaissance progress in capital province (totaling 100% progress in capital province)
+10 Prestige
Set global flag: "atlantean_renaissance"

---

Rebirth of Atlantis
Fires only once for a rebirthed Atlantean Empire, if formed before the previous event had a chance to fire.

Conditions:
-Is Atlantean Empire tag
-Renaissance is enabled
-Does not have global flag: "atlantean_renaissance"

Mean Time to Happen: 1 year
No modifiers needed

"Our rise as the rightful rulers of all of Atlantis has been a godsend for the artistic communities of the continent. With a central authority estabilished and some semblance of stability in the future, prospectual artists no longer fear starving as they put aside their artistic tools for farming ones. As [Root.Monarch.GetName]'s influence grows, so does our need to have all sorts of glorious mementos; a ruler's artistic legacy cannot be underestimated. At first, it was the maintenance of palaces and creation of new places of worship, but now, we aren't satisfied until those places look absolutely gorgeous, and commissioning the finest paintings and statues we can muster is the logical next step."

Only Option: "It seems we're experiencing a cultural rebirth!"
+50% Renaissance progress in capital area (including provinces owned by other countries)
+50% Renaissance progress in capital province (totaling 100% progress in capital province)
+10 Prestige
Set global flag: "atlantean_renaissance"

**Comment by Zephyrum** [comment]
Artworks in Antillia

Fires only once. Nation who meets the MTTH reduction requisites are more likely, but not guaranteed, to get it.

Conditions:
-Capital is in Antillia
-Did not embrace the Renaissance
-EITHER OF: An Atlantean country has the Renaissance embraced, or can see an European province
-EITHER OF: Is at least 1475 or has unified Antillia into a single country

MTTH: 10 years
Reduced by:
-Unifying Antillia
-Has a province with at least 25 development
-Has a level 3 CoT (requires Dharma)

"Our isolationism has been shattering quickly as foreigners start to realize Antillia's existence, in it's secluded corner of the ocean. Contacts from the North have spoken about a golden age of art and philosophy in their homelands, and, needless to say, Antillian clergymen feel jealous. Despite our tradition of clerical rule, Antillia has not had grand churches built to last forever and impress all visitors, a travesty that tarnishes our reputation. We must seek to equate ourselves to these foreigners in cultural development if our legacy is to be permanent."

Only Option: "We'll have the grandest of churces built!"
+100% Renaissance progress in capital province
+10 prestige

**Comment by Zephyrum** [comment]
Colonialism:
Won't need an event.
TL;DR - the Atlanteans and Antillians are essentially already colonists in a new land, so there is no reason they should bother chasing the institution.
Instead, we edit the institution gain conditions in the main file and add the following modifiers:
Is Atlantean: +2% monthly spread (fully enabled in 50 months or about 4 years)
Is Antillian: +1.5% monthly spread (fully enabled in 70 months or about 6 years)
Conditions for them is having the capital be in those regions.

**Comment by Zephyrum** [comment]
The Press
I feel like at this point, any major flavor would go in too lightly-touched, so it's better to have a simple solution by adding a spread modifier again.
For Heptarchics in Atlantis and Antillia: gain the same spread modifiers Protestants and Reformeds get.

Worst case scenario, the 15 diplo tech also gives you passive spread wherever you are on the globe.
